src/content/docs/nrql/get-started/introduction-nrql-new-relics-query-language.mdx

```diff
@@ -208,6 +208,113 @@ You can also use NRQL with our [NerdGraph API](/docs/apis/graphql-api/tutorials/
+  [<a href="/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#sel-until">UNTIL</a> time]
+  [<a href="/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#sel-timezone">WITH TIMEZONE</a> timezone]
+  [<a href="/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#sel-compare">COMPARE WITH</a> time]
+  [<a href="/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#sel-timeseries">TIMESERIES</a> time]
+```
+
+NRQL's basic rules are as follows:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        <DoNotTranslate>**NRQL rule**</DoNotTranslate>
+      </th>
+
+      <th>
+        Details
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        Required values
+      </td>
+
+      <td>
+        The [`SELECT`](#state-select) clause and [`FROM`](#sel-from) clause are required. All other clauses are optional. You can start your query with either `SELECT` or `FROM`.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Query string size
+      </td>
+
+      <td>
+        The query string must be less than 4 KB.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Case sensitivity
+      </td>
+
+      <td>
+        * The data type names and attribute names are case sensitive.
+        * [NRQL clauses](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#clauses) and [functions](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#functions) are not case sensitive.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Syntax for strings
+      </td>
+
+      <td>
+        NRQL uses single quotes to designate strings. For example:
+
+        ```sql
+        ... where traceId = '030a573f0df02c57'
+        ```
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Non-standard custom event and attribute names
+      </td>
+
+      <td>
+        Events that we report by default have names that contain alphanumeric characters, colons (`:`), and underscores (`_`). Attribute names can have those characters and periods (`.`). Default-reported names start with a letter. Custom names that don't follow these guidelines must be enclosed with backticks in NRQL queries. For example:
+
+        ```sql
+        ... FACET `Logged-in user`
+        ```
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Data type coercion
+      </td>
+
+      <td>
+        We don't support data type "coercion." For more information, see [Data type conversion](/docs/insights/nrql-new-relic-query-language/nrql-reference/nrql-syntax-components-functions#type-conversion).
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+These are the basic rules for NRQL, and we have more information in our [NRQL reference](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/) to help you build your queries.
```

Comment:
I'm not so sure you need to restate that these are the basic rules.

Reply:
Agreed and changed